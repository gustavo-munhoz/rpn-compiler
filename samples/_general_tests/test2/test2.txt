(17.5 4 %)
((1 RES) MEM)
(50 (2 RES) /)
((MEM) (-2 3 ^) +)
((1000 6.25 |) MEM)
(((MEM) (3 RES) %) (2 RES) *)
(0.25 0.5 *)
(1000 (1 RES) *)
(1 (7 RES) |)
((MEM) (1 RES) +)
(99 -6.5 %)
((1 RES) MEM)
(MEM)