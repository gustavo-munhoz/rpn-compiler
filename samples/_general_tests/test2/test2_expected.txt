(17.5 4 %)                    => 1.5 (0x3E00)
((1 RES) MEM)                 => 1.5 (0x3E00)
(50 (2 RES) /)                => 33 (0x5020)
((MEM) (-2 3 ^) +)            => -6.5 (0xC680)
((1000 6.25 |) MEM)           => 160 (0x5900)
(((MEM) (3 RES) %) (2 RES) *) => -182 (0xD9B0)
(0.25 0.5 *)                  => 0.125 (0x3000)
(1000 (1 RES) *)              => 125 (0x57D0)
(1 (7 RES) |)                 => 0.66650390625 (0x3955)
((MEM) (1 RES) +)             => 160.625 (0x5905)
(99 -6.5 %)                   => 1.5 (0xC500)
((1 RES) MEM)                 => 1.5 (0xC500)
(MEM)                         => 1.5 (0xC500)